tests/p2p/bootstrap_test/config.yaml

```diff
@@ -10,7 +10,7 @@ deployment_ready_time_out: 120 #default 120
 config_path: '../config.toml'
 
 bootstrap:
-  image: 'spacemeshos/go-spacemesh-dev:go_auto'
+  image: 'spacemeshos/go-spacemesh-dev:develop'
```

Comment:
go-spacemesh:develop

Reply:
fixed in 207fd37bd289dcc691ce4b4207a102bc0f009af7